### DISCUSSION TITLE
Widget based on filtered or searchable table

### DISCUSSION THREAD

**Discussion by davidfrigolet** [filamentphp/filament#2140] [Help]
Hi community,

We would like to display a widget on top of a table which draws statistics based on the data being viewed in the table. If we filter or search in the table the information in the widget would be updated. Is this possible with Filament?

Thanks in advance

**Comment by davidfrigolet** [comment]
It's possible?

Thanks

**Comment by electronick86** [comment]
Would be very useful indeed.
